src/core/ddsi/src/q_plist.c

```diff
@@ -521,50 +562,74 @@ static dds_return_t deser_generic (void * __restrict dst, size_t * __restrict ds
   return DDS_RETCODE_BAD_PARAMETER;
 }
 
-static size_t ser_generic_size (const void *src, size_t srcoff, const enum pserop * __restrict desc)
+static void ser_generic_size_partial (size_t *dstoff, const void *src, size_t srcoff, const enum pserop * __restrict desc)
```

Comment:
I wouldn't call this "partial": the only other "partial" is in "fini" and there it is actually about doing part of the type description. This is "embeddable" or something ...